style.css

```diff
@@ -0,0 +1,292 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
```

Comment:
Please use an other kind of font size except "Poppins" , It is not required

Reply:
- [ ] Kindly explain more about these suggested changes. It is not clear because I can't even see the `font size` you mentioned in your comment. Please make sure to deeply these suggested changes.